src/bundle/Resources/public/js/scripts/fieldType/base/base-rich-text.js

```diff
@@ -5,7 +5,12 @@
         constructor() {
             this.ezNamespace = 'http://ez.no/namespaces/ezpublish5/xhtml5/edit';
             this.xhtmlNamespace = 'http://www.w3.org/1999/xhtml';
-            this.customTags = Object.keys(global.eZ.adminUiConfig.richTextCustomTags);
+            this.customTags = Object.keys(global.eZ.adminUiConfig.richTextCustomTags).filter(
```

Comment:
`customTags` might be confusing, maybe `notInlineCustomTags` or `blockCustomTags`?

Reply:
But now I see that in other files we have terminology inlineCustomTag/customTag, so OK. :)